Entities/ACrab.h

```diff
@@ -515,6 +515,18 @@ int FirearmActivationDelay() const;
 	void SetLimbPathPushForce(float force);
 
 
+	/// <summary>
+	/// Gets this ACrab's Stride Sound. Ownership is not transferred!
```

Comment:
This is a bit persnickety of me, but for method comments we only capitalize and use code naming for object types. So ACrab or Attachable would be right, but Stride Sound wouldn't be. In short, this should be `Gets this ACrab's stride sound. Ownership is NOT tranferred!`